### DISCUSSION TITLE
How do I change fonts on all pages?

### DISCUSSION THREAD

**Discussion by ktatar** [alshedivat/al-folio#2505] [Q&A] [answered]
Hello,

I would like to choose and try different fonts for different text classes. How is this possible? 

Thank you,

**Comment by george-gca** [comment]
I'd recommended reading and watching the video [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Web_fonts), where they explain a lot about setting fonts and how to test them in your site using the dev tools (also present on chrome-based browsers). As for where to set, probably you'll end up setting them in [_sass/_base.scss](https://github.com/alshedivat/al-folio/blob/master/_sass/_base.scss).

**Comment by ktatar** [comment]
Thanks for the links. Do I have to add font-family to all items in the _base.scss ? Is there a way to set the fonts generally? I think only 2-3 fonts are used in the template.

I think there is a way to add font-family variable to _variable.scss but I couldn't make it work yet.

**Reply by george-gca** [reply]
You can add it to the outermost element in the part that you want it to be. If it is sitewide, then this would be in the `body` element.

**Reply by ktatar** [reply]
Sure, that works. What if it was a manual font though? That is, if I want to use a specific font with on a file in the repo? I couldn't get that to work yet. An example would be so good.

**Reply by george-gca** [reply, marked as answer]
Check [this answer on stackoverflow](https://stackoverflow.com/a/38086507).
